src/main/java/org/gridsuite/modification/server/modifications/VscModification.java

```diff
@@ -257,6 +265,11 @@ private List<ReportNode> hvdcAngleDroopActivePowerControlAdder(HvdcLine hvdcLine
         return reports;
     }
 
+    private static NetworkModificationException modifyHvdcAngleDroopActivePowerControl(@lombok.NonNull String equipementName, @lombok.NonNull String attributeName) {
+        throw new NetworkModificationException(NetworkModificationException.Type.WRONG_HVDC_ANGLE_DROOP_ACTIVE_POWER_CONTROL,
```

Comment:
return instead of throw, it can make code coverage test better, I created a PR: https://github.com/gridsuite/network-modification-server/pull/515